stacktrace: index out of range

```
./stmps -mpris
panic: runtime error: index out of range [-1] [recovered]
	panic: runtime error: index out of range [-1]

goroutine 1 [running]:
github.com/rivo/tview.(*Application).Run.func1()
	/home/spezifisch/go/pkg/mod/github.com/rivo/tview@v0.0.0-20231031172508-2dfe06011790/application.go:251 +0x45
panic({0x958d00?, 0xc0009b8d38?})
	/usr/lib/go/src/runtime/panic.go:770 +0x132
main.(*queueData).GetCell(0xc000b26000?, 0x26?, 0x1?)
	/home/spezifisch/Projects/stmps/page_queue.go:159 +0x385
github.com/rivo/tview.(*Table).MouseHandler.func1(0xda68?, 0x18?, 0xc000b24000?)
	/home/spezifisch/go/pkg/mod/github.com/rivo/tview@v0.0.0-20231031172508-2dfe06011790/table.go:1652 +0xb7
github.com/rivo/tview.(*Table).MouseHandler.(*Box).WrapMouseHandler.func2(0x6000?, 0xc00003b9d0?, 0x414765?)
	/home/spezifisch/go/pkg/mod/github.com/rivo/tview@v0.0.0-20231031172508-2dfe06011790/box.go:209 +0x58
github.com/rivo/tview.(*Flex).MouseHandler.func1(0x3, 0xc0009336e0, 0xc00090d0c0)
	/home/spezifisch/go/pkg/mod/github.com/rivo/tview@v0.0.0-20231031172508-2dfe06011790/flex.go:239 +0xc3
github.com/rivo/tview.(*Flex).MouseHandler.(*Box).WrapMouseHandler.func2(0x20c0?, 0x414765?, 0x18?)
	/home/spezifisch/go/pkg/mod/github.com/rivo/tview@v0.0.0-20231031172508-2dfe06011790/box.go:209 +0x58
github.com/rivo/tview.(*Pages).MouseHandler.func1(0x3, 0xc0009336e0, 0xc00090d0c0)
	/home/spezifisch/go/pkg/mod/github.com/rivo/tview@v0.0.0-20231031172508-2dfe06011790/pages.go:294 +0xca
github.com/rivo/tview.(*Pages).MouseHandler.(*Box).WrapMouseHandler.func2(0x3830?, 0x10?, 0xc000100008?)
	/home/spezifisch/go/pkg/mod/github.com/rivo/tview@v0.0.0-20231031172508-2dfe06011790/box.go:209 +0x58
github.com/rivo/tview.(*Flex).MouseHandler.func1(0x3, 0xc0009336e0, 0xc00090d0c0)
	/home/spezifisch/go/pkg/mod/github.com/rivo/tview@v0.0.0-20231031172508-2dfe06011790/flex.go:239 +0xc3
github.com/rivo/tview.(*Flex).MouseHandler.(*Box).WrapMouseHandler.func2(0x2660?, 0x4421ee?, 0x1?)
	/home/spezifisch/go/pkg/mod/github.com/rivo/tview@v0.0.0-20231031172508-2dfe06011790/box.go:209 +0x58
github.com/rivo/tview.(*Application).fireMouseActions.func1(0xffb1?)
	/home/spezifisch/go/pkg/mod/github.com/rivo/tview@v0.0.0-20231031172508-2dfe06011790/application.go:438 +0x182
github.com/rivo/tview.(*Application).fireMouseActions(0xc00037a0e0, 0xc0009336e0)
	/home/spezifisch/go/pkg/mod/github.com/rivo/tview@v0.0.0-20231031172508-2dfe06011790/application.go:475 +0x2cd
github.com/rivo/tview.(*Application).Run(0xc00037a0e0)
	/home/spezifisch/go/pkg/mod/github.com/rivo/tview@v0.0.0-20231031172508-2dfe06011790/application.go:374 +0x86a
main.(*Ui).Run(0xc000360240)
	/home/spezifisch/Projects/stmps/gui.go:183 +0xa5
main.main()
	/home/spezifisch/Projects/stmps/stmps.go:125 +0x82c
```

--- a/page_queue.go
+++ b/page_queue.go
@@ -153,7 +153,7 @@ func (q *QueuePage) updateQueue() {
 
 // queueData methods, used by tview to lazily render the table
 func (q *queueData) GetCell(row, column int) *tview.TableCell {
-	if row >= len(q.playerQueue) || column >= queueDataColumns {
+	if row >= len(q.playerQueue) || column >= queueDataColumns || row < 0 || column < 0 {
 		return nil
 	}
 	song := q.playerQueue[row]

